src/auth/mysql_native_password.ts

```diff
@@ -1,20 +1,19 @@
-import { encode } from "../../deps.ts";
-import { Hash } from "https://deno.land/x/checksum@1.1.0/mod.ts";
+import { encode } from "../../deps.ts"
+import { Hash } from "https://deno.land/x/checksum@1.1.0/mod.ts"
```

Comment:
Whether to use a semicolon or not This is a faction controversy, the controversy is not meaningful, but I am willing to use `deno fmt` to standardize the code style

Reply:
I can do that with the next commit.